lib/apiservers/service/restapi/handlers/vch_create.go

```diff
@@ -398,9 +398,9 @@ func buildCreate(op trace.Operation, d *data.Data, finder *find.Finder, vch *mod
 					OpsPassword: &opsPassword,
 				}
 			}
-			if err := c.OpsCredentials.ProcessOpsCredentials(true, c.Target.User, c.Target.Password); err != nil {
-				return nil, util.WrapError(400, err)
-			}
+		}
+		if err := c.OpsCredentials.ProcessOpsCredentials(true, c.Target.User, c.Target.Password); err != nil {
+			return nil, util.WrapError(400, err)
```

Comment:
We should use constants from the `net/http` pkg instead of hard numbers.

Reply:
@anchal-agrawal these are being fixed in a separate PR.